docker-images/dockerfiles/serving-tensorrt.dockerfile

```diff
@@ -14,15 +14,16 @@ COPY script/run_server.sh /script/run_server.sh
 
 RUN . /etc/os-release && \
     apt-get update && \
-    apt-get install --no-install-recommends -y wget ca-certificates && \
-    wget "https://developer.download.nvidia.com/compute/cuda/repos/$ID$(echo $VERSION_ID | tr -d .)/x86_64/7fa2af80.pub" -O /etc/apt/trusted.gpg.d/cuda.asc && \
+    apt-get install --no-install-recommends -y \
```

Comment:
Better to merge these two lines:
`
	apt-get install --no-install-recommends -y \
		gnupg curl software-properties-common && \
`
into one line.